cmd/acr/purge.go

```diff
@@ -52,10 +52,10 @@ const (
 )
 
 var (
-	defaultPoolSize          = runtime.GOMAXPROCS(0)
-	defaultRepoBatchSize     = int32(100)
-	repoBatchSizeDescription = "Number of repositories queried at once"
-	concurrencyDescription   = fmt.Sprintf("Number of concurrent purge tasks. Range: [1 - %d]", maxPoolSize)
+	defaultPoolSize         = runtime.GOMAXPROCS(0)
+	defaultPageBatchSize    = int32(100)
```

Comment:
nit: Do we want to rename this as well to align with the flag name `repoPageSize`?

Reply:
Good point. Done.